c10/cuda/CUDAStream.cpp

```diff
@@ -174,12 +174,14 @@ static void initGlobalStreamState() {
   num_gpus = device_count();
   // Check if the number of GPUs matches the expected compile-time max number
   // of GPUs.
+#ifdef FBCODE_CAFFE2
```

Comment:
Why?

Reply:
For non-fbcode the check is always true, and Clang++ warns about that.